packages/plugin-server-session/session.js

```diff
@@ -29,16 +42,14 @@ module.exports = {
 }
 
 const sendSessionSummary = client => sessionCounts => {
-  const releaseStage = inferReleaseStage(client)
-
   // exit early if the reports should not be sent on the current releaseStage
-  if (isArray(client.config.notifyReleaseStages) && !includes(client.config.notifyReleaseStages, releaseStage)) {
-    client._logger.warn(`Session not sent due to releaseStage/notifyReleaseStages configuration`)
+  if (isArray(client._config.enabledReleaseStages) && !includes(client._config.enabledReleaseStages, client._config.releaseStage)) {
```

Comment:
An empty array should mean all release stages are enabled - there are a couple of other places throughout the codebase that will also require changing

Reply:
Ah yes this logic was corrected somewhere but appears in a few other different places. Will address it.